manifests/tasks/test-model-rest-svc-task.yaml

```diff
@@ -25,8 +25,10 @@ spec:
 
         echo "\nCall service:"
 
+        export DEPLOYMENT_NAME=`sed 's/[^a-z0-9.-]/-/g; s/-*$//' <<< $(params.model-name)`-$(params.model-version)
```

Comment:
In this task I would name it as a `SERVICE_NAME` since we are hitting a service endpoint here that just happens to have the same name as the deployment.

Reply:
Okay I think all of this sounds good. Adding a task for sanitising the name is not what I had in mind first but I think that is a good solution.